package.json

```diff
@@ -4,28 +4,26 @@
   "repository": "https://github.com/CatalaLang/catala-dsfr",
   "scripts": {
     "clean": "rescript clean -with-deps",
-    "build": "yarn run pre && vite build",
+    "build": "yarn run pre && yarn run assets && vite build",
     "deploy": "yarn run build --base=/demos/catala/ && rsync -rv --delete-before dist/*",
     "serve": "vite preview",
     "dev": "yarn run pre && vite",
```

Comment:
Shouldn't this also include `yarn run assets` ?

Reply:
Yes, I'll add it in the `pre` command :+1: